content/04-postgresql/07-create-and-delete-databases-and-tables.mdx

```diff
@@ -561,3 +561,63 @@ This will remove the database and any child elements, like tables, within.
 This article covered the basics of how to create and delete databases and tables within PostgreSQL.  These are some of the most basic commands required to set up a database system and being defining the structure of your data.
 
 As mentioned earlier, the SQL statements covered in this PostgreSQL tutorial, particularly the `CREATE TABLE` statement, have many additional parameters can be used to change PostgreSQL's behavior.  You can find out more about these by checking out the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/sql-createtable.html).
+
+## FAQ
+
+
+<details><summary>Does PostgreSQL support the `IF NOT EXISTS` clause when using the `CREATE DATABASE` command?</summary>
+
+Yes, PostgreSQL supports the use of `IF NOT EXISTS` when creating both databases and tables. The below demonstrates using the clause for table creation, and you can read more about its use [here](https://www.prisma.io/dataguide/postgresql/create-and-delete-databases-and-tables#create-tables-conditionally-with-the-if-not-exists-clause) 
```

Comment:
Usually, it's best if the link text is more descriptive than "here" so that the reader knows where they're being directed.